WISE-Community/WISE-Client
# feat(Authoring): Allow adding branch from add step button

## Changes
- Added a create branch view
- Added an edit branch view

## Test
- Make sure you can create a branch
- Make sure you can edit a branch

Closes #1745
